.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - 10.15.3
+cache:
+  directories:
+  - node_modules
+script:
+  - npm run build
```

Comment:
Awesome work here. Don't you think it would be best to add the `npm run build` command before the `npm test`?

Reply:
That command `npm run build ` is good if our application is hosted because it increases performance by bundling the static files together such that only one request is sent other than making multiple requests for each file. For the case of running tests, it slows down the process for the first time, however, I will go ahead and include it. Thanks for that initiative.